$NetBSD$

Use sh instead of Bash, as the script seems to be bashism-free.

--- preload.init.in.orig	2008-10-23 01:08:20.000000000 +0000
+++ preload.init.in
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 #	@initddir@/@PACKAGE@
 #
